integreat_cms/cms/views/pages/page_form_view.py

```diff
@@ -254,6 +254,21 @@ def post(self, request, *args, **kwargs):
                 page_translation_form.instance.page = page_form.save()
             # Save page translation form
             page_translation_form.save()
+            # If any source translation changes to draft, set all depending translations/versions to draft
+            if (
+                "status" in page_translation_form.changed_data
+                and page_translation_form.cleaned_data.get("status") == status.DRAFT
+            ):
+                language_tree_node = region.language_tree_nodes.get(
+                    language_id=language.id
+                )
+                if not language_tree_node.is_leaf():
+                    PageTranslation.objects.filter(
+                        page_id=page_translation_form.instance.page_id,
+                        language_id__in=[
+                            node.id for node in language_tree_node.get_descendants()
+                        ],
+                    ).update(status=page_translation_form.cleaned_data.get("status"))
```

Comment:
```suggestion
            if page_translation_form.instance.status == status.DRAFT:
                language_tree_node = region.language_node_by_id.get(language.id)
                if not language_tree_node.is_leaf():
                    partial_language_tree = LanguageTreeNode.get_tree(
                        parent=language_tree_node
                    )
                    page_translation_form.instance.page.translations.filter(
                        language__in=[node.language for node in partial_language_tree]
                    ).update(status=status.DRAFT)
```
To fix the problems mentioned above, I suggest the following changes.